### DISCUSSION TITLE
Is there a place for moving up, moving down, and deleting configuration shortcuts

### DISCUSSION THREAD

**Discussion by chf-gh** [codex-team/editor.js#2877] [Q&A]
Can I do the Move Up, Move down, delete configuration shortcut, I can't find it

**Comment by neSpecc** [comment]
They are internal tools, but it is a nice feature. How I see the solution:

We [have](https://github.com/codex-team/editor.js/blob/8f365f006328a36faacf60352e20a0fc365c227b/src/components/tools/base.ts#L43) a support for `shortcut` public getter in all types of tools. You can add a "Command+Option+ArrowUp" shortcut for "Move Up" tune, etc inside the Editor code. If it won't work right after that, you'll need to update BlockEvents.ts or UI.ts module to add support for shortcuts. You should use `shortcuts.ts` util for that
